src/core/a-entity.js

```diff
@@ -241,21 +240,11 @@ var proto = Object.create(ANode.prototype, {
 
       if (this.hasLoaded) { return; }
 
-      // Attach to parent object3D.
-      this.addToParent();
-
-      // Scene load.
-      function sceneLoadCallback () { self.updateComponents(); }
-      if (this.isScene) {
-        ANode.prototype.load.call(this, sceneLoadCallback);
```

Comment:
The scene should call ANode.load *without* the `isEntity` filter. We want the scene to wait on *all* chk.d Nodes (including `<a-assets>`, whereas entities need only wait on all child entities.